src/server/game/Entities/Player/Player.cpp

```diff
@@ -16807,19 +16807,29 @@ QuestGiverStatus Player::GetQuestDialogStatus(Object* questgiver)
         {
             if (CanSeeStartQuest(quest))
             {
-                if (SatisfyQuestLevel(quest, false))
-                {
-                    if (quest->IsAutoComplete())
-                        result2 = DIALOG_STATUS_REWARD_REP;
-                    else if (getLevel() <= (GetQuestLevel(quest) + sWorld->getIntConfig(CONFIG_QUEST_LOW_LEVEL_HIDE_DIFF)))
-                    {
-                        if (quest->IsDaily())
-                            result2 = DIALOG_STATUS_AVAILABLE_REP;
+                if (SatisfyQuestLevel(quest, false)) {
+                    bool bLowLevel = (getLevel() > (GetQuestLevel(quest) + sWorld->getIntConfig(CONFIG_QUEST_LOW_LEVEL_HIDE_DIFF)));
```

Comment:
I'm not sure if this is intended to be hungarian notation or supposed to sound like "below level". Either way, I would recommend changing it.
```suggestion
                    bool isBelowLevel = (getLevel() > (GetQuestLevel(quest) + sWorld->getIntConfig(CONFIG_QUEST_LOW_LEVEL_HIDE_DIFF)));
```

Reply:
It's hungarian notation (boolean), but it's just a habit, we can make it whatever conforms most to the project's coding standards.

Although what do you mean by "changing its uses as well"? It's just a flag to indicate if it's a low level quest or not.

Reply:
Sorry for not being more clear, by changing it's uses I just meant to also rename where it's being used. My suggested change had only been where it was declared.

Reply:
I see. I'm fine either way, but I think "isBelowLevel" is a little misleading. A low level quest, by definition, is below your level, it's just a question of how much.

Once you pass the threshold defined in the worldserver.conf file, the quest is considered "low level" and will no longer show up in game or on the mini map unless explicitly tracked. The relevant configuration options are as follows...

```
#
#    Quests.LowLevelHideDiff
#        Description: Level difference between player and quest level at which quests are
#                     considered low-level and are not shown via exclamation mark (!) at quest
#                     givers.
#        Default:     4  - (Enabled, Hide quests that have 4 levels less than the character)
#                     -1 - (Disabled, Show all available quest marks)

Quests.LowLevelHideDiff = 4

#
#    Quests.HighLevelHideDiff
#        Description: Level difference between player and quest level at which quests are
#                     considered high-level and are not shown via exclamation mark (!) at quest
#                     givers.
#        Default:     7  - (Enabled, Hide quests that have 7 levels more than the character)
#                     -1 - (Disabled, Show all available quest marks)

Quests.HighLevelHideDiff = 7
```

So, once the character's level is higher than the level of the quest plus the value defined by `Quests.LowLevelHideDiff `, it is considered "low level".

As far as naming convention goes, I'm fine with `isLowLevel`, but `isBelowLevel` seems to miss the mark.

Reply:
The name you suggest is fine with me as well. Thanks for the pull request 😃

Reply:
OK, the change is in place. Still pretty new to GitHub, though, so please let me know if anything is off, thanks.

Reply:
ty sdyess